lti_consumer/plugin/compat.py

```diff
@@ -94,3 +94,12 @@ def publish_grade(block, user, score, possible, only_if_higher=False, score_dele
         score_deleted=score_deleted,
         grader_response=comment,
     )
+
+
+def has_access(*args, **kwargs):
+    """
+    Import and run the `has_access` method from courseware module.
```

Comment:
I feel an example of how can this function be used for example what all parameters can be passed if we include that it will save a lof of time in future to find the ussage.

Reply:
@farhaanbukhsh The ``compat.py`` is just an abstraction to help mock platform API calls during testing. This function doesn't contain any logic itself. I think it is better to force the user to check platform documentation. Otherwise, it will have duplicate documentation for the same function, and add the overhead of maintaining the docstring if anything changes in platform python API. You can check ``compat.py`` and see that most functions are doing the same thing, calling platform function.

Reply:
The point of maintaining the docstring seems legit.